Steer/DigitizerWorkflow/src/FT0DigitizerSpec.cxx

```diff
@@ -101,15 +101,16 @@ class FT0DPLDigitizerTask : public o2::base::BaseDPLDigitizer
           // call actual digitization procedure
           mDigitizer.setEventID(part.entryID);
           mDigitizer.setSrcID(part.sourceID);
-          mDigitizer.process(&hits, mDigitsBC, mDigitsCh, labels);
+          mDigitizer.process(&hits, mDigitsBC, mDigitsCh, mDigitsTrig, labels);
         }
       }
     }
-    mDigitizer.flush_all(mDigitsBC, mDigitsCh, labels);
+    mDigitizer.flush_all(mDigitsBC, mDigitsCh, mDigitsTrig, labels);
 
     // send out to next stage
     pc.outputs().snapshot(Output{"FT0", "DIGITSBC", 0, Lifetime::Timeframe}, mDigitsBC);
     pc.outputs().snapshot(Output{"FT0", "DIGITSCH", 0, Lifetime::Timeframe}, mDigitsCh);
+    pc.outputs().snapshot(Output{"FT0", "DIGITSTRIGGER", 0, Lifetime::Timeframe}, mDigitsTrig);
```

Comment:
I think it is better if all detectors name it "TRIGGERINPUT", i.e. ``pc.outputs().snapshot(Output{"FT0", "TRIGGERINPUT", 0...``

Reply:
OK, thank you. This is what I wanted to ask. I will rename it